release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -3009,3 +3009,19 @@ This update contains changes from Kubernetes 1.20.11. More information can be fo
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-7-39"]
+=== RHBA-2021:4885 - {product-title} 4.7.39 bug fix update
+
+Issued: 2021-12-08
+
+{product-title} release 4.7.39 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4885[RHBA-2021:4885] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:4884[RHBA-2021:4884] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6564591[{product-title} 4.7.39 container image list]
+
+[id="ocp-4-7-39-upgrading"]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
@darshan-nagaraj should 4.7 be replaced with {product-version}?

Reply:
As chris mentioned, it will stay static.